frontend/Dockerfile

```diff
@@ -1,12 +1,10 @@
-FROM node:20.7.0-alpine
+FROM node:20-alpine
```

Comment:
@MukuFlash03 is there a reason we removed the semver here? I understand that we need to upgrade node, but we can do so by bumping up the version, right?

Reply:
So, we aren't removing semver, it's already a part of the base image and upgraded to the latest [version 7.5.4](https://www.npmjs.com/package/semver).

I know that during the previous PR, we upgraded the base image which contained the latest semver version.
However, I did go through the list of [available node images ](https://hub.docker.com/_/node/tags?page=1&name=alpine) and saw that the numeric base versions (20, 21) are being updated with vulnerability fixes pretty often and these version were newer than say (20.7.0).

We could use some newer versions (21.5, 21, 20.10.0) but I saw that version 20 had the same latest package / module versions as these.

Reply:
semver = semantic versioning as a concept, not the semver package.
I understand that you still have `node:20` but you have removed the `.7.0` part.
I would suggest using the full semantic version - e.g. 

```
FROM node:20.10.0-alpine
```

Using 20-alpine is like "latest", you don't know which version of 20 you have.
Also, there is no reason that I am aware of to *not* upgrade to node:21.